File: eg/99bottles.pl
#!/usr/bin/perl

$bottles = 9999;
while ($bottles > 0) {
	print "$bottles bottles of beer on the wall,\n";
	print "$bottles bottles of beer,\n";
	print "Take none down, pass none around,\n";
	$bottles = $bottles - 1;
	print "$bottles bottles of beer on the wall.\n";
	print "\n";
}

File: eg/99bottles.lua
#!/usr/bin/env lua

bottles = 9999
while (bottles > 0) do
	print(bottles, "bottles of beer on the wall,")
	print(bottles, "bottles of beer,")
	print("Take none down, pass none around,")
	print(bottles - 1, "bottles of beer on the wall.");
	print("");
	bottles = bottles - 1
end
